src/DssSpell.sol

```diff
@@ -47,17 +87,118 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
+        // IS-001 - 348,452.30 DAI - 0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4 https://github.com/makerdao/mips/pull/463/files
+        // Rounded up from 348,452.30 to 348,453
+        DssExecLib.sendPaymentFromSurplusBuffer(ISCU_WALLET,  348_453);
+        // RWF-001 - 2,055,000 DAI - 0x96d7b01Cc25B141520C717fa369844d34FF116ec https://mips.makerdao.com/mips/details/MIP40c3SP61#transactions
+        DssExecLib.sendPaymentFromSurplusBuffer(RWF_WALLET,  2_055_000);
+
+        // VEST.restrict( Only recipient can request funds
+        //     VEST.create(
+        //         Recipient of vest,
+        //         Total token amount of vest over period,
+        //         Start timestamp of vest,
+        //         Duration of the vesting period (in seconds),
+        //         Length of cliff period (in seconds),
+        //         Manager address
+        //     )
+        // );
+
+        // Core Unit DAI Budget Streams
+        // GOV-001 | 2022-04-01 to 2023-04-01 | 1,079,793 DAI | 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73 https://forum.makerdao.com/t/mip40c3-sp59-govalpha-budget-2022-23/13144
+        MCD_VEST_DAI.restrict(
+            MCD_VEST_DAI.create(
+                GOV_WALLET_1,
+                1_079_793 * WAD,
+                APR_01_2022,
+                ONE_YEAR,
+                0,
+                GOV_WALLET_1
+            )
+        );
+        // IS-001 | 2022-03-01 to 2022-08-01 | 700,356.90 DAI | 0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4 https://github.com/makerdao/mips/pull/463/files
+        MCD_VEST_DAI.restrict(
+            MCD_VEST_DAI.create(
+                ISCU_WALLET,
+                ISCU_DAI_STREAM_AMOUNT,
+                MAR_01_2022,
+                FIVE_MONTHS,
+                0,
+                ISCU_WALLET
+            )
+        );
+        // RWF-001 | 2022-04-01 to 2022-12-31 | 6,165,000 DAI | 0x96d7b01Cc25B141520C717fa369844d34FF116ec https://mips.makerdao.com/mips/details/MIP40c3SP61#transactions
+        MCD_VEST_DAI.restrict(
+            MCD_VEST_DAI.create(
+                RWF_WALLET,
+                6_165_000 * WAD,
+                APR_01_2022,
+                EIGHT_MONTHS,
+                0,
+                RWF_WALLET
+            )
+        );
+        // Remove/Revoke Stream #27 (RWF-001) on DssVestSuckable https://mips.makerdao.com/mips/details/MIP40c3SP61#transactions
+        MCD_VEST_DAI.yank(27);
+
+        // Gelato Keeper Network DAI Budget Stream
+        // https://mips.makerdao.com/mips/details/MIP63c4SP3
+        // Address: 0x926c21602fec84d6d0fa6450b40edba595b5c6e4
+        // Amount: 1,000 DAI/day
+        // Start Date: Apr 1, 2022
+        // End Date: Oct 1, 2022
+        MCD_VEST_DAI.restrict(
+            MCD_VEST_DAI.create(
+                GELATO_WALLET,
+                183_000 * WAD,
+                APR_01_2022,
+                SIX_MONTHS,
+                0,
+                GELATO_WALLET
```

Comment:
Same comment here.  Not sure we should give this contract the ability to `yank()`.  It's not explicitly in the code and I don't see an upgradeable pattern.

```suggestion
                address(0)
```

Reply:
beat you to it ;-) https://github.com/makerdao/spells-mainnet/pull/235#discussion_r840676359